Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -503,7 +503,8 @@ void ChangeDisc(const std::string& newFileName)
 		auto sizeofpath = fileName.find_last_of("/\\") + 1;
 		if (fileName.substr(sizeofpath).length() > 40)
 		{
-			PanicAlert("Saving iso filename to .dtm failed; max file name length is 40 characters.");
+			PanicAlertT("Saving disc image filename to .dtm failed.\n"
+			            "The maximum filename length is 40 characters.");
```

Comment:
Maybe "The filename of the game's disc image must be less than 40 characters long.", to make it clear which file we're talking about.